code/datums/perks/fate.dm

```diff
@@ -0,0 +1,197 @@
+/datum/perk/paper_worm
+	name = "Paper Worm"
+	desc = "You always look at the bright side of life but seems there's something you'd forgotten" 
+	icon_state = "paper"
+
+/datum/perk/paper_worm/assign(mob/living/carbon/human/H)
+	..()
+	holder.sanity.positive_prob += 20
+
+/datum/perk/paper_worm/remove()
+	holder.sanity.positive_prob -= 20
+	..()
+
+/datum/perk/freelancer
+	name = "Freelancer"
+	icon_state = "skills"
+	desc = "You're a jack of all trades but master of none"
+
+/datum/perk/freelancer/assign(mob/living/carbon/human/H)
+	..()
+	var/maxstatindex
+	for (var/i=1 to holder.stats.stat_list.len-1)
+		if(holder.stats.getStat(holder.stats.stat_list[i]) > holder.stats.getStat(holder.stats.stat_list[i+1]))
+			maxstatindex = i
+		else
+			maxstatindex = i+1
+	for(var/name in holder.stats.stat_list)
+		if(name != holder.stats.stat_list[maxstatindex])
+			holder.stats.addTempStat(name, 4, INFINITY, "Fate Freelancer")
+		else
+			holder.stats.addTempStat(name, -10, INFINITY, "Fate Freelancer")
+
+/datum/perk/freelancer/remove()
+	holder.stats.removeTempStat(STAT_MEC, "Fate Freelancer")
+	holder.stats.removeTempStat(STAT_COG, "Fate Freelancer")
+	holder.stats.removeTempStat(STAT_BIO, "Fate Freelancer")
+	holder.stats.removeTempStat(STAT_ROB, "Fate Freelancer")
+	holder.stats.removeTempStat(STAT_TGH, "Fate Freelancer")
+	holder.stats.removeTempStat(STAT_VIG, "Fate Freelancer")
+	..()
+
+/datum/perk/nihilist
+	name = "Nihilist"
+	desc= "You have lost faith in this world and in its people."
+	icon_state = "eye" //https://game-icons.net/1x1/lorc/tear-tracks.html
+
+/datum/perk/nihilist/assign(mob/living/carbon/human/H)
+	..()
+	holder.sanity.positive_prob += 10
+	holder.sanity.negative_prob += 20
+
+/datum/perk/nihilist/remove()
+	holder.sanity.positive_prob -= 10
+	holder.sanity.negative_prob -= 20
+	holder.stats.removeTempStat(STAT_COG, "Fate Nihilist")
+	..()
+
+/datum/perk/moralist
+	name = "Moralist"
+	icon_state = "moralist" //https://game-icons.net/
+	desc = "You are good human being. Love life and life will love you back. Love people and they will love you back."
```

Comment:
"You are a good human being. Love life, and life will love you back. Love people, and they will love you back."